lib/fhir_client/sections/crud.rb

```diff
@@ -147,12 +147,12 @@ def base_create(resource, options, format)
         options[:format] = format
         reply = post resource_url(options), resource, fhir_headers(options)
         if [200, 201].include? reply.code
-          type = reply.response[:headers][:content_type]
+          type = reply.response[:headers].select{|x| x.downcase=='content-type'}['content-type']
```

Comment:
If you're needing to downcase the hash keys ... I don't think the last piece of this will work:

```ruby
irb(main):011:0> headers = { 'Content-Type' => 'application/fhir+json', 'other-header' => 'other-value' }
=> {"Content-Type"=>"application/fhir+json", "other-header"=>"other-value"}
irb(main):012:0> headers.select{|x| x.downcase=='content-type'}
=> {"Content-Type"=>"application/fhir+json"}
irb(main):013:0> headers.select{|x| x.downcase=='content-type'}['content-type']
=> nil
```

Some tests might be nice. 😄